docs/index.src.html

```diff
@@ -3442,6 +3437,20 @@ <h1 id="api-macro">Macro</h1>
 </code></pre>
     </p>
 
+    <p id="phpstan-support">We provide a PHPStan extension out of the box so it can understand macros and mixins.</p>
+
+    <p>If you're using Laravel, you should consider using <a href="https://github.com/nunomaduro/larastan">larastan</a>
+        which provides a complete support of Laravel features in PHPStan (including Carbon macros).
+        Else you can install <strong>phpstan/extension-installer</strong> if you don't have it:</p>
```

Comment:
This installation method is pretty much unusable for me and many others, see https://github.com/phpstan/extension-installer/issues/17

How about putting the non-magical installation method first (even before hinting to use larastan)?

```html
Enable the extension by adding it to your `phpstan.neon` configuration:

<pre>code>
includes:
- vendor/nesbot/carbon/extension.neon
</pre></code>
```

Reply:
I will change that.